src/components/pages/ChooseAccount/index.js

```diff
@@ -1,16 +1,90 @@
 import React, { PureComponent } from 'react';
-import { View, TouchableOpacity, Text } from 'react-native';
+import { View, TouchableOpacity, Text, ScrollView, Image, FlatList} from 'react-native';
+import StyledView from '../../atoms/StyledView';
+import StyledText from '../../atoms/StyledText';
+import StyledButton from '../../atoms/StyledButton';
 
 export default class ChooseAccount extends PureComponent {
+  state = {
+    selectAccount: false,
+    data:[{
+      'imgUrl' :'https://i.imgur.com/K3KJ3w4h.jpg',
+      'bankName': 'HSBC Bank (UK)',
+      'accountType':'Advance Account',
+      'sortCode':'20-01-89',
+      'accountNumber': '54923765'
+    },
+    {
+      'imgUrl' :'https://i.imgur.com/K3KJ3w4h.jpg',
+      'bankName': 'HSBC Bank (UK)',
+      'accountType':'Premier Account',
+      'sortCode':'20-01-89',
+      'accountNumber': '54923766'
+    },
+    {
+      'imgUrl' :'https://i.imgur.com/K3KJ3w4h.jpg',
+      'bankName': 'HSBC Bank (UK)',
+      'accountType':'`Corporate` Account',
+      'sortCode':'20-01-89',
+      'accountNumber': '54923767'
+    }]
+  }
+
+  selectAccount =(account) => 
+    this.setState({
+      selectAccount:account
+    });
+  
+
+renderItem = ({item}) => {
+  const bdColor = item.accountNumber === this.state.selectAccount ? 'blue': 'lightgray';
+  const bgColor = item.accountNumber === this.state.selectAccount ? '#ccebff': '#fff';
+    return( 
+      <TouchableOpacity onPress={ () => this.selectAccount(item.accountNumber)}>
+        <View style={{flex:1, flexDirection:'row',justifyContent:"center", alignItems:"center", padding: 20, margin:10,borderColor: bdColor, backgroundColor:bgColor, borderStyle:"solid", borderWidth:1
```

Comment:
because we have a fair amount of inline styling, is it worth creating a styles const above the class?